### PR TITLE
Add category filters to library PAY-1710 PAY-1711

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Adds category filter pills to library and hooks them up to tracks tab

fitleripgif

### CODE REVIEW

File: packages/web/src/pages/saved-page/components/desktop/LibraryCategorySelectionMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const dispatch = useDispatch()
+  const selectedCategory = useSelector(getSelectedCategory)
+  const handleClick = (value: LibraryCategoryType) => {
+    dispatch(setSelectedCategory(value))
+  }
+
+  return (
+    <div role='radiogroup' className={styles.container}>
+      {CATEGORIES.map((c) => (
+        <HarmonySelectablePill
```
Comment (nicoback2) [inline packages/web/src/pages/saved-page/components/desktop/LibraryCategorySelectionMenu.tsx:44]: Planning to add a SelectablePillRadioGroup to Harmony + allow specifying HarmonySelectablePill element type as radio input. It's not the most straightforward so gonna do it as a project follow-on most likely.

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--add-filters-library

File: packages/common/src/store/pages/saved-page/reducer.ts [hunk 2]
```diff
@@ -27,16 +30,17 @@ const initialState = {
   initialFetch: false,
   hasReachedEnd: false,
   fetchingMore: false,
-  tracks: initialLineupState
-}
+  tracks: initialLineupState,
+  selectedCategory: 'favorite'
```
Comment (Kyle-Shanks) [inline packages/common/src/store/pages/saved-page/reducer.ts:34]: should the enum be used here?

**Review by Kyle-Shanks — APPROVED** [review]
Nice! also whats up with the styles in the video? lol

**Comment by nicoback2** [comment]
> Nice! also whats up with the styles in the video? lol

omg lol i just noticed that, i think maybe the vid quality is extremely low so it didnt get some of the color details

File: packages/common/src/store/pages/saved-page/reducer.ts (hunk 2 above)
Comment (nicoback2) [reply packages/common/src/store/pages/saved-page/reducer.ts:34]: ah yes thanks!

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--add-filters-library

**Pull request merged**